packages/angular/package.json

```diff
@@ -12,6 +12,11 @@
     "directory": "packages/angular"
   },
   "type": "module",
+  "peerDependencies": {
```

Comment:
I think there should be no such peerDependencies?

PeerDependencies should be updated in the file `packages/angular/src/package.json`?

<img width="596" height="606" alt="Image" src="https://github.com/user-attachments/assets/75a1b20a-6b90-4c7e-90e7-52515ddbef1e" />